### PR TITLE
SWC-6746: just render paragraphs when not editing

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java [hunk 1]
```diff
@@ -182,17 +201,6 @@ public UserProfileEditorWidgetViewImpl(
     locationTextBox = locationSuggestBox.getTextBox();
     locationTextBox.addStyleName("form-control");
     locationSuggestBoxContainer.add(locationSuggestBox);
-    // note, not adding email since it's not editable here.
-    textBoxes =
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java:186]: Previously, this code would run through this array of text boxes and set them to readOnly if we were not editing.  Now, there are "renderer" (static, read-only) elements that are shown instead.

File: src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java [hunk 2]
```diff
@@ -215,6 +223,7 @@ public Widget asWidget() {
   @Override
   public void setUsername(String username) {
     this.username.setText(username);
+    usernameRenderer.setText(username);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java:226]: Initialize both the editor and renderer with the current value

File: src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java [hunk 3]
```diff
@@ -395,33 +409,46 @@ public void addKeyDownHandlerToFields(KeyDownHandler keyDownHandler) {
...
     currentPosition.setPlaceholder(isEditing ? "Enter current position" : "");
+    industry.setVisible(isEditing);
+    industryRenderer.setVisible(!isEditing);
     industry.setPlaceholder(isEditing ? "Enter industry/discipline" : "");
 
     editProfileButton.setVisible(!isEditing);
     saveProfileButton.setVisible(isEditing);
     cancelButton.setVisible(isEditing);
-    linkRenderer.setVisible(!isEditing);
-    link.setVisible(isEditing);
```
Comment (jay-hodgson) [inline src/main/java/org/sagebionetworks/web/client/widget/profile/UserProfileEditorWidgetViewImpl.java:422]: Moved up (next to where we set the placeholder) in the same method

**Review by hallieswan — APPROVED** [review]
Looks good to me!

**Pull request merged**